pkg/deploy/generator/resources_rp.go

```diff
@@ -576,19 +589,43 @@ cat >/etc/td-agent-bit/td-agent-bit.conf <<'EOF'
 	Port 29230
 EOF
 
-export AZURE_CLOUD_NAME=$AZURECLOUDNAME
+echo "FLUENTBITIMAGE=$FLUENTBITIMAGE" >/etc/sysconfig/fluentbit
 
-az login -i --allow-no-subscriptions
-az account set -s "$SUBSCRIPTIONID"
+cat >/etc/systemd/system/fluentbit.service <<'EOF'
+[Unit]
+After=docker.service
+Requires=docker.service
+StartLimitIntervalSec=0
 
-systemctl start docker.service
-az acr login --name "$(sed -e 's|.*/||' <<<"$ACRRESOURCEID")"
+[Service]
+RestartSec=1s
+EnvironmentFile=/etc/sysconfig/fluentbit
+ExecStartPre=-/usr/bin/docker rm -f %N
+ExecStartPre=-/usr/bin/docker pull $FLUENTBITIMAGE
+ExecStartPre=-/bin/mkdir -p /var/lib/fluent
```

Comment:
These are already done by the steps above, are they needed here?

Reply:
Removed from service unit.